pandas/core/reshape/pivot.py

```diff
@@ -102,8 +102,9 @@ def pivot_table(
         on the rows and columns.
     dropna : bool, default True
         Do not include columns whose entries are all NaN. If True,
-        rows with a NaN value in any column will be omitted before
-        computing margins.
+        - rows with a NaN value in any column will be omitted before computing margins,
+        - index/column keys containing NA values will be dropped (see ``dropna``
+        parameter in ``pandas.DataFrame.groupby``).
```

Comment:
You'll also need to update the entry in `shared_docs` within `pandas.core.frame`.

Reply:
Can you use `an NA value` instead of `a NaN value`. Also link to the groupby via `` :meth:`DataFrame.groupby` `` instead of ``pandas.DataFrame.groupby``.

Reply:
It's `:meth:` and not `:method:` for sphinx.